src/Tooltip.js

```diff
@@ -173,16 +174,34 @@ class Tooltip extends React.Component {
   }
 
   addTargetEvents() {
-    this._target.addEventListener('mouseover', this.onMouseOverTooltip, true);
-    this._target.addEventListener('mouseout', this.onMouseLeaveTooltip, true);
-    this._target.addEventListener('keydown', this.onEscKeyDown, true);
-    this._target.addEventListener('focusin', this.show, true);
-    this._target.addEventListener('focusout', this.hide, true);
-
-
-    ['click', 'touchstart'].forEach(event =>
-      document.addEventListener(event, this.handleDocumentClick, true)
-    );
+    if (this.props.trigger) {
+      let triggers = this.props.trigger.split(' ');
+      if (!triggers.includes('manual')) {
```

Comment:
`Array.prototype.includes` is not supported by IE. Rather than requiring the users to add a polyfill, can you use `Array.prototype.indexOf` can check that it is `-1`. Similar for the usages below (but checking that it is `-1`)